contentcuration/contentcuration/view/internal_views.py

```diff
@@ -51,11 +52,11 @@ def check_version(request):
     version = json.loads(request.body)['version']
     status = None
 
-    if str(version) >= VERSION_OK[0]:
+    if cmp(LooseVersion(version), LooseVersion(VERSION_OK[0])) >= 0:
```

Comment:
You can just use the operators inline right? So:

`LooseVersion(...) >= LooseVersion(...)`